clang/include/clang/Basic/DiagnosticSerializationKinds.td

```diff
@@ -24,8 +24,9 @@ def err_fe_ast_file_modified : Error<
     DefaultFatal;
 def err_fe_pch_file_overridden : Error<
     "file '%0' from the precompiled header has been overridden">;
-def note_pch_required_by : Note<"'%0' required by '%1'">;
-def note_pch_rebuild_required : Note<"please rebuild precompiled header '%0'">;
+def note_ast_file_required_by : Note<"'%0' required by '%1'">;
+def note_ast_file_rebuild_required
+    : Note<"please rebuild precompiled file '%0'">;
```

Comment:
Do we use the "precompiled file" term anywhere else? If we're trying to unify and clarify the naming, why not use "AST file"? Alternatively, we could be even more specific like we already are in `err_fe_ast_file_modified`.

Reply:
Ok, in that case I'll defer to @vsapsai who was trying to get some unification of terminology in diagnostics in https://github.com/llvm/llvm-project/pull/101413.